scripts/branch.sh

```diff
@@ -0,0 +1,7 @@
+branch_desc=$1;
+branch_name="feature-${branch_desc}";
+
+git checkout master
+git pull
+git checkout -b $branch_name
+git push --set-upstream origin $branch_name
```

Comment:
this is just a helper script meant to support the branch naming structure we seem to be using